vsintegration/tests/unittests/Resources.MockTypeProviders/DummyProviderForLanguageServiceTesting/TypeProviderEmit.fsi

```diff
@@ -1,289 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I've replaced the TypeProviderEmit.fs by the latest ``ProvidedTypes.fs`` which has support for static method parameters